portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,33 +42,45 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     List<String> comments = new ArrayList<>();
+    if (quantity == 0){ quantity = 10; } //Default set at max of 10 comments shown upon loading screen
```

Comment:
```suggestion
    if (quantity == 0) { quantity = 10; }  // Default set at max of 10 comments shown upon loading screen
```
Some style issues with spacing. We should consider adding a java linter in the future to avoid these sort of nitpicky comments.